package.json

```diff
@@ -34,7 +34,7 @@
   "dependencies": {
     "@alertlogic/al-aws-collector-js": "4.1.2",
     "datadog-lambda-js": "2.23.0",
-    "async": "3.1.0",
+    "async": "3.2.2",
```

Comment:
we need to make paws-collector/package.json change as a separate PR. And once it is shipped we need to bump paws-collector dependency version in all collectors.